mti_nma/steps/singlecell/singlecell_utils.py

```diff
@@ -0,0 +1,127 @@
+import numpy as np
+import pandas as pd
+import labkey as lk
+from lkaccess import LabKey as lka
+from skimage import transform as sktrans
+
+def query_data_from_labkey(CellLineId):
```

Comment:
Could you add docstrings for both of these functions? I've been using a template style for docstrings if you want it to keep consistent or you can do whatever you like.

Reply:
Will do.